app/src/main/java/ch/epfl/sweng/swenggolf/profile/EditProfileActivity.java

```diff
@@ -24,31 +24,37 @@ protected void onCreate(Bundle savedInstanceState) {
 
         user = Config.getUser();
         if (user != null) {
-            EditText editText = findViewById(R.id.edit_name);
-            String userName = user.getUserName();
-            editText.setText(userName);
-            editText.setSelection(userName.length());
-
+            displayElement((EditText) findViewById(R.id.edit_name), user.getUserName());
+            displayElement((EditText) findViewById(R.id.edit_pref), user.getPreference());
             ImageView imageView = findViewById(R.id.ivProfile);
             ProfileActivity.displayPicture(imageView, user, this);
         }
```

Comment:
If user is null here, something went really wrong. It means the user could get to this part of the app without being logged in Firebase. 
We could do something like that :
```suggestion
        }
        else{
             throw new NullPointerException("The user is not logged here");
        }
```

Reply:
Yes, you're right! Or maybe it should be an assert?